py/janitor/site/templates/layout.html

```diff
@@ -2,18 +2,21 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-        <title>Janitor -
+        <title>Janitor
+            {% if self.page_title() | trim | length > 0 %}-
             {% block page_title %}
             {% endblock page_title %}
+            {% endif %}
         </title>
+        <link rel="author" title="About these documents" href="/about" />
         <link rel="stylesheet" href="/_static/alabaster.css" type="text/css" />
         <link rel="stylesheet" href="/_static/lintian.css" type="text/css" />
-        <link rel="author" title="About these documents" href="/about" />
         <link rel="stylesheet" href="/_static/typeahead.css" type="text/css" />
         <link rel="stylesheet" href="/_static/pygments.css" type="text/css" />
         <link rel="stylesheet" href="/_static/datatables.css" type="text/css" />
         <meta name="viewport"
               content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />
+        {% include "head.html" ignore missing without context %}
```

Comment:
There is no head.html file - why is this necessary?

Reply:
Yup. Just like there isn't a `{% include "analytics.html" ignore missing without context %}`
REF: https://github.com/jelmer/janitor/blob/7b73eeeade65ff2434b2a17f579e0fe1cf225ef3/py/janitor/site/templates/layout.html#L28C1-L28C70 

The reason for it, is to allow for custom CSS to be included.
This came about due to https://github.com/jelmer/janitor/pull/949/

Reply:
For what its worth, Kali has dropped its own site instance - now basing it from `janitor.site.simple`
This is because as Kali's site code wasn't kept up-to-date with whats upstream for a number of years.
The goal is to be easier to maintain going fortward.